### ISSUE TITLE
MeetingReadinessChecker - invalid video input + deprecated API call

### ISSUE THREAD

**Issue by chuckntaylor** [aws/amazon-chime-sdk-js#2236]
### What happened and what did you expect to happen?

Working on creating a device checker based on the meetingReadinessChecker demo provided. When running the demo application, I get two issues. (Screen shots attached).
I get an `Invalid video input device` Error that gets triggered during the `videoTest()` - specifically on line 246 of [meetingReadinessChecker.ts](https://github.com/aws/amazon-chime-sdk-js/blob/main/demos/browser/app/meetingReadinessChecker/meetingReadinessChecker.ts)

```ts
const videoInputResp = await this.meetingReadinessChecker.checkVideoInput(videoInput)
```
<img width="1582" alt="Screen Shot 2022-05-17 at 10 22 36 AM" src="https://user-images.githubusercontent.com/17050325/168844350-217bfab0-91f5-450a-9633-e734a65dae13.png">

The second issue I am seeing is in regards to line 322

```ts
const networkUdpResp = await this.meetingReadinessChecker.checkNetworkUDPConnectivity();
```

Here, I am getting a deprecation warning to not use `getRTCPeerConnectionStats()` and instead use the new `clientMetricReport.getRTCStatsReport()` returned by `metricsDidReceive(clientMetricReport)` callback. This warning appears to be coming from the internals of `checkNetworkUDPConnectivity`.

<img width="1595" alt="Screen Shot 2022-05-17 at 10 22 54 AM" src="https://user-images.githubusercontent.com/17050325/168844364-ee92eaf0-540e-4b6e-8b37-6ca7086aa927.png">

Lastly, when the tests are done, I see that my camera is still active. Does something need to be called at the end to turn off the camera and the microphone after testing completes? I seem to recall in version 2, this happened on its own when testing was complete.

### Have you reviewed our existing documentation?

- [X] GitHub [issues](https://github.com/aws/amazon-chime-sdk-js/issues)
- [X] FAQs [page](https://aws.github.io/amazon-chime-sdk-js/modules/faqs.html)
- [X] README [page](https://github.com/aws/amazon-chime-sdk-js#amazon-chime-sdk-for-javascript)
- [X] Feature-specific [guides](https://github.com/aws/amazon-chime-sdk-js#guides)

### Reproduction steps

Run the provided meetingReadinessChecker demo application.

My device:
macOS 12.3.1
MacBook Pro - Apple M1 Max 64GB
Camera Model Id: FaceTime HD Camera

### Amazon Chime SDK for JavaScript version

amazon-chime-sdk-js@3.2.0

### What browsers are you seeing the problem on?

Chrome 101, Firefox 100

### Browser version

Chrome 101, Firefox 100

### Browser console logs

In image

**Comment by ltrung** [comment]
@chuckntaylor Thanks for reporting. The first error `Invalid video input device` is a miss in our v3 where we should use `stopVideoInput` instead of `startVideoInput(null)` to stop the camera stream. The second one is also another miss on our side that we did not replace the deprecated APIs. I will work out on a fix for this.

**Comment by chuckntaylor** [comment]
@ltrung thanks for taking a look at this! Let me know if you need any more information.

**Comment by ltrung** [comment]
@chuckntaylor The fix is merged. Please let me know if it fixes all the issues for you.

**Comment by ltrung** [comment]
@chuckntaylor It will be released to npm in the next 3.3.0 release.

**Comment by ltrung** [comment]
3.3.0 has been released in NPM so close this issue.
